kafka-connect-neo4j/src/main/resources/kafka-connect-sink.properties

```diff
@@ -0,0 +1,31 @@
+##
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+
+neo4j.server.uri=Type: String;\nDescription: The Bolt URI
```

Comment:
should we note a default URL (do we have one?)

Reply:
Sure I'll add it